.github/workflows/test.yml

```
@@ -0,0 +1,45 @@
+    
+    - name: Run type checking
+      run: npm run check
+    
+    - name: Run unit tests
+      run: npm test
+    
+    - name: Generate coverage report
+      run: npm run test:coverage
+    
+    - name: Upload coverage to Codecov
+      uses: codecov/codecov-action@v3
```

actionlint (1.7.7)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)